### DISCUSSION TITLE
Inserire etichette su isoipse (Contours) Raster

### DISCUSSION THREAD

**Discussion by Riccardocasoli** [opendatasicilia/HfcQGIS-md#86] [Domande] [answered]
RItengo fenomenale la possibilità di visualizzare molto semplicemente un DTM o qualsiasi layer raster che rappresenta una superficie come Contours. Ma non riesco a rappresentare l'etichetta che sarebbe necessaria sulle curve di livello principali (Index contours). é possibile in qualche maniera?  Grazie. Riccardo

**Comment by pigreco** [comment, marked as answer]
Ciao Riccardo e grazie per la domanda.
Quello che hai realizzato è solo una tematizzazione del dato raster, attualmente questa modalità non permette la visualizzazione delle etichette.

Se desideri visualizzare le curve di livello e le relative etichette occorre:
1. estrarre le curve di livello come vettori usando il tool, da Processing, Curve di Livello;
2. il layer delle curve appena estratte contiene l'attributo delle quote delle curve, quindi puoi usarlo per tematizzare.

qui un esempio:
https://pigrecoinfinito.com/2018/12/05/qgis-etichettare-le-curve-di-livello/

saluti

**Comment by Riccardocasoli** [comment]
Grazie Salvatore!
il metodo da te indicato è quello classico ma richiede la generazione di un tematismo vettoriale (e si fa agevolmente ogniqualvolta vi sia la necessità). Ma mi chiedevo se con le "magiche" funzioni di Qgis si riesca a mettere un'etichetta sulla linea "Index contour" generata mediante tematizzazione del Raster. Mi pare di aver capito che con la versione attuale di Qgis non sia possibile. ...spero in un prossimo futuro. Grazie e ciao Riccardo

**Comment by Riccardocasoli** [comment]
Simile problema l'avevo avuto volendo rappresentare il cono di depressione di alcuni pozzi in base al dato di abbassamento contenuto nella tabella di uno shp puntuale ("Pozzi") mediante costruzione di geometria multipla (cerchi) attorno al punto con spaziatura non lineare ma logaritmica (tramite funzioni di Qgis). C'ero riuscito! ma non sono riuscito a mettere i numerini sui singoli cerchi che rappresentano curve di isoabbassamento attorno al pozzo! Scusa la digressione e grazie per la tua dedizione alla divulgazione e alla conoscenza di Qgis e delle sue potenti funzioni! Ciao.
 Riccardo

**Comment by Riccardocasoli** [comment]
Ciao Totò
    mi permetto di disturbarti per porti un quesito tecnico
      sull'etichettatura in Qgis.
    Come si possono Etichettare i singoli vertici di una polilinea 3D
      con la quota dei vertici stessi?
    Ho provato a cercare sui tuoi fantastici esempi ma non sono
      riuscito.
    Ho provato generando una geometria nella finestra delle etichette
      (finestra di "Posizionamento -> Generatore geometria ->
      ***@***.***)") e in Valore etichetta ho scritto
      ***@***.***)" .....ma nulla anche con altri tentativi
      ***@***.***").
    Mi pare di aver capito che la funzione "Nodes_to_points()" non
      generi un punto 3D dal momento che ho provato ad etichettare la
      coordinata $x e me l'ha mostrata correttamente in corrispondenza
      di ogni vertice!!
    
    
    Come fare? 
    Hai qualche suggerimento?
    Grazie in anticipo.
    Riccardo
     
    
      
      
      
      
      
      
	
	
      Riccardo
              Casoli,
          Geologo (cell: 3289026131)
      
      
         
          
            
               
            
          
          
            
              STUDIO TECNICO
            
          
          
            
              Riccardo Casoli
              
              via Coda, 22/E - 13900
                  Biella Chiavazza
                Tel e fax: 01523346
                e-mail: ***@***.***
                PEC: ***@***.***
            
          
          
            
              C.F.: CSLRCR68L14E379M 
                P.IVA: IT02746620026

**Reply by pigreco** [reply]
Ciao Riccardo, per favore apri altra Discussions con titolo adatta e se puoi allega un piccolo dataset
